src/module-elasticsuite-catalog/Model/ResourceModel/Setup/PropertyMapper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Smile ElasticSuite to newer
+ * versions in the future.
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteCatalog
+ * @author   Benjamin Rosenberger <bensch.rosenberger@gmail.com>
+ * @copyright 2023 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+namespace Smile\ElasticsuiteCatalog\Model\ResourceModel\Setup;
+
+use Magento\Eav\Model\Entity\Setup\PropertyMapperAbstract;
+
+/**
+ * Elasticsearch catalog EAV attribute mappings.
+ *
+ * @category Smile
+ * @package  Smile\ElasticsuiteCatalog
+ * @author   Benjamin Rosenberger <bensch.rosenberger@gmail.com>
+ */
+class PropertyMapper extends PropertyMapperAbstract
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function map(array $input, $entityTypeId)
+    {
+        return [
+            'is_displayed_in_autocomplete' => $this->_getValue($input, 'is_displayed_in_autocomplete', 0),
+            'is_used_in_spellcheck' => $this->_getValue($input, 'is_used_in_spellcheck', 0),
+            'facet_min_coverage_rate' => $this->_getValue($input, 'facet_min_coverage_rate', 90),
+            'facet_max_size' => $this->_getValue($input, 'facet_max_size', 10),
+            'facet_sort_order' => $this->_getValue($input, 'facet_sort_order', 25),
```

Comment:
should be `Smile\ElasticsuiteCore\Search\Request\BucketInterface::SORT_ORDER_COUNT` not `25`